inc/Styles/Component.php

```diff
@@ -74,7 +74,7 @@ public function get_slug() : string {
 	public function initialize() {
 		add_action( 'wp_enqueue_scripts', array( $this, 'action_enqueue_styles' ) );
 		add_action( 'wp_head', array( $this, 'action_preload_styles' ) );
-		add_action( 'enqueue_block_editor_assets', array( $this, 'action_enqueue_block_editor_styles' ) );
+		add_action( 'admin_init', array( $this, 'action_add_editor_styles' ) );
```

Comment:
I'm curious why you chose `admin_init` here. Typically, `add_editor_style()` is called on `after_setup_theme`.

Reply:
I have probably looked at [this example](https://developer.wordpress.org/reference/functions/add_editor_style/#comment-1171). Changed to `after_setup_theme` following your recommendations.